azurerm/resource_arm_dev_test_lab_schedule.go

```diff
@@ -0,0 +1,489 @@
+	client := meta.(*ArmClient).devTestLabSchedulesClient
+	ctx := meta.(*ArmClient).StopContext
+
+	id, err := parseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	resGroup := id.ResourceGroup
+	devTestLabName := id.Path["labs"]
+	name := id.Path["schedules"]
+
+	resp, err := client.Get(ctx, resGroup, devTestLabName, name, "")
+
+	if err != nil {
+		if utils.ResponseWasNotFound(resp.Response) {
+			d.SetId("")
+			return nil
+		}
+		return fmt.Errorf("Error making Read request on Dev Test Lab Schedule %s: %s", name, err)
+	}
+
+	d.Set("name", resp.Name)
+	if location := resp.Location; location != nil {
+		d.Set("location", azure.NormalizeLocation(*location))
+	}
+	d.Set("dev_test_lab_name", devTestLabName)
+	d.Set("resource_group_name", resGroup)
+
+	if props := resp.ScheduleProperties; props != nil {
+		if timeZoneId := props.TimeZoneID; *timeZoneId != "" {
+			d.Set("time_zone_id", *timeZoneId)
+		}
+
+		switch status := props.Status; status {
+		case dtl.EnableStatusEnabled:
+			d.Set("status", string(dtl.EnableStatusEnabled))
+		case dtl.EnableStatusDisabled:
+			d.Set("status", string(dtl.EnableStatusDisabled))
+		default:
+		}
+
+		if weeklyRecurrence := props.WeeklyRecurrence; weeklyRecurrence != nil {
+			if err := d.Set("weekly_recurrence", flattenAzureRmDevTestLabScheduleRecurrenceWeekly(props.WeeklyRecurrence)); err != nil {
+				return fmt.Errorf("Error setting `weeklyRecurrence`: %#v", err)
+			}
+		}
+
+		if dailyRecurrence := props.DailyRecurrence; dailyRecurrence != nil {
+			if err := d.Set("daily_recurrence", flattenAzureRmDevTestLabScheduleRecurrenceDaily(props.DailyRecurrence)); err != nil {
+				return fmt.Errorf("Error setting `dailyRecurrence`: %#v", err)
+			}
+		}
+
+		if hourlyRecurrence := props.HourlyRecurrence; hourlyRecurrence != nil {
+			if err := d.Set("hourly_recurrence", flattenAzureRmDevTestLabScheduleRecurrenceHourly(props.HourlyRecurrence)); err != nil {
+				return fmt.Errorf("Error setting `dailyRecurrence`: %#v", err)
+			}
+		}
+
+		if err := d.Set("notification_settings", flattenAzureRmDevTestLabScheduleNotificationSettings(props.NotificationSettings)); err != nil {
+			return fmt.Errorf("Error setting `notificationSettings`: %#v", err)
+		}
+	}
+
+	flattenAndSetTags(d, resp.Tags)
+
+	return nil
+}
+
+func resourceArmDevTestLabSchedulesDelete(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).vmExtensionClient
+	ctx := meta.(*ArmClient).StopContext
+
+	id, err := parseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	resGroup := id.ResourceGroup
+	name := id.Path["schedules"]
+	devTestLabName := id.Path["labs"]
+
+	future, err := client.Delete(ctx, resGroup, devTestLabName, name)
+	if err != nil {
+		return err
+	}
+
+	return future.WaitForCompletionRef(ctx, client.Client)
+}
+
+func expandArmDevTestLabScheduleRecurrenceDaily(d *schema.ResourceData) (*dtl.DayDetails, error) {
+	dailyRecurrenceConfigs := d.Get("daily_recurrence").(*schema.Set).List()
```

Comment:
as this is now a List we'll need to make this:

```suggestion
	dailyRecurrenceConfigs := d.Get("daily_recurrence").([]interface{})
```

Reply:
Done.